CHANGELOG.md

```diff
@@ -4,6 +4,68 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
+## [Preview Release 4.0.0-preview1] - 2021-08-20
+
+### Added
+- Added [MSSQLSERVER_42109](https://docs.microsoft.com/en-us/sql/relational-databases/errors-events/mssqlserver-42109-database-engine-error?view=sql-server-ver15) and [MSSQLSERVER_42108](https://docs.microsoft.com/en-us/sql/relational-databases/errors-events/mssqlserver-42108-database-engine-error?view=sql-server-ver15) to retriable transient error list.
+[#1215](https://github.com/dotnet/SqlClient/pull/1215)
+- `PoolBlockingPeriod` is enabled for .NET Standard.
+[#1181](https://github.com/dotnet/SqlClient/pull/1181)
+- `SqlDataReader.GetColumnSchema()` API is now exposed in .NET Standard.
+[#1181](https://github.com/dotnet/SqlClient/pull/1181)
+- Added new App Context Switch `UseSystemDefaultSecureProtocols` to go back to Operating System's supported protocols.
+[#1168](https://github.com/dotnet/SqlClient/pull/1168)
+- Added missing component model annotations to `SqlConnectionStringBuilder`, `SqlConnection`, `SqlCommand`, `SqlParameter` and `SqlDataAdapter`
+[#1152](https://github.com/dotnet/SqlClient/pull/1152)
+- The new public API `DisableOutputParameters` is added to `SqlCommand`.
```

Comment:
The name is actually EnableOptimizedParameterBinding - I know because I came up with it! 😀

Reply:
And it's the name that's the important part right? ;)